src/game/player.py

```diff
@@ -1,19 +1,27 @@
 class Player:
-    def __init__(self):
+    def __init__(self, name='unnamed'):
```

Comment:
What's the purpose of the default parameter? In what context do we want an unnamed Player?